content/en/docs/concepts/services-networking/dual-stack.md

```diff
@@ -68,47 +72,162 @@ An example of an IPv6 CIDR: `fdXY:IJKL:MNOP:15::/64` (this shows the format but
+
+When you define a Service you can optionally configure it as dual stack. To specify the behavior you want, you
+set the `.spec.ipFamilyPolicy` field to one of the following values:
+
+   * `SingleStack`: Single-stack service. The control plane allocates a cluster IP for the Service, using the first configured service cluster IP range.
+   * `PreferDualStack`: 
+      * Only used if the cluster has dual-stack enabled. Allocates IPv4 and IPv6 cluster IPs for the Service
+      * If the cluster does not have dual-stack enabled, this setting follows the same behavior as `SingleStack`.
+   * `RequireDualStack`: Allocates Service `.spec.ClusterIPs` from both IPv4 and IPv6 address ranges 
+      * Selects the `.spec.ClusterIP` from the list of `.spec.ClusterIPs` based on the address family of the first element in the `.spec.ipFamilies` array. The cluster must have dual-stack networking configured.
+
+If you would like to define which IP family to use for single stack or define the order of IP families for dual-stack, you can choose the address families by setting an optional field, `.spec.ipFamilies`, on the Service. 
 
 {{< note >}}
-The default address family for your cluster is the address family of the first service cluster IP range configured via the `--service-cluster-ip-range` flag to the kube-controller-manager.
+The `.spec.ipFamilies` field is immutable because the `.spec.ClusterIP` cannot be reallocated on a Service that already exists. If you want to change `.spec.ipFamilies`, delete and recreate the Service.
 {{< /note >}}
 
-You can set `.spec.ipFamily` to either:
+You can set `.spec.ipFamilies` to any of the following array values:
+
+- `["IPv4"]`
+- `["IPv6"]`
+- `["IPv4","IPv6"]` (dual stack)
+- `["IPv6","IPv4"]` (dual stack)
 
-   * `IPv4`: The API server will assign an IP from a `service-cluster-ip-range` that is `ipv4`
-   * `IPv6`: The API server will assign an IP from a `service-cluster-ip-range` that is `ipv6`
+The first family you list is used for the legacy `.spec.ClusterIP` field.
 
-The following Service specification does not include the `ipFamily` field. Kubernetes will assign an IP address (also known as a "cluster IP") from the first configured `service-cluster-ip-range` to this Service.
+### Dual-stack Service configuration scenarios
+
+These examples demonstrate the behavior of various dual-stack Service configuration scenarios.
+
+#### Dual-stack options on new Services
+
+   1. This Service specification does not explicitly define `.spec.ipFamilyPolicy`. When you create this Service, Kubernetes assigns a cluster IP for the Service from the first configured `service-cluster-ip-range` and sets the `.spec.ipFamilyPolicy` to `SingleStack`. ([Services without selectors](/docs/concepts/services-networking/service/#services-without-selectors) and [headless Services](/docs/concepts/services-networking/service/#headless-services) with selectors will behave in this same way.)
```

Comment:
You can remove the indentation of this list.
If you want a numbered list, you can use:

1. item 1
1. item 2
1. item 3

Reply:
The source for that Markdown would look like
```markdown
1. item 1
1. item 2
1. item 3
```

Reply:
Enough elements between that 1. 1. 1. doesn't work in this case.